Tasks/WindowsMachineFileCopy/task.loc.json

```diff
@@ -55,12 +55,12 @@
       }
     },
     {
-      "name": "MachineFilter",
+      "name": "MachineNames",
       "type": "string",
-      "label": "ms-resource:loc.input.label.MachineFilter",
+      "label": "ms-resource:loc.input.label.MachineNames",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "ms-resource:loc.input.help.MachineFilter"
+      "helpMarkDown": "ms-resource:loc.input.help.MachineNames"
```

Comment:
I have checked this. There are no such changes in WindowsCopy, AzureCopy and RunPowershell task.